### PR TITLE
Adds D3 Simple Slider

### PR DESCRIPTION
**Pull request by DanielMwendwa**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -73,6 +73,7 @@
     <main id="main-container"></main>
     <script src="vendor/d3-6.3.1/d3.min.js"></script>
     <script src="vendor/d3-legend-2.25.6/d3-legend.min.js"></script>
+    <script src="https://unpkg.com/d3-simple-slider"></script>
```
Comment (IsaackMwenda) [inline public/index.html]: we should host this locally under vendor/

File: public/js/controllers/traffic_graphs_controller.js [hunk 2]
```diff
@@ -208,6 +208,42 @@ export class TrafficGraphsController {
             yLimitSentFiltered = d3.max(tenMinGraphFilteredData, d => d.total_sent),
             yLimitFailed = d3.max(dailyFailedTotal, d => d.total_errored),
             yLimitFailedFiltered = d3.max(tenMinGraphFilteredData, d => d.total_errored); 
+        
+        // Add Y axis Slider
+        total_received_sms_graph
+            .append("g")
+            .attr("class", "receivedSlider")
```
Comment (IsaackMwenda) [inline public/js/controllers/traffic_graphs_controller.js:215]: the font colour for the slider values is quite faint. Lets make it more visible.

File: public/js/controllers/traffic_graphs_controller.js [hunk 3]
```diff
@@ -1301,6 +1353,18 @@ export class TrafficGraphsController {
             }
         });
 
+        // Draw sent graph with user-selected y-axis limit
```
Comment (IsaackMwenda) [inline public/js/controllers/traffic_graphs_controller.js:1356]: update on failed graph too.

**Review by IsaackMwenda — APPROVED** [review]

File: public/index.html (hunk 1 above)
Comment (DanielMwendwa) [reply public/index.html]: Updated, thanks.

File: public/js/controllers/traffic_graphs_controller.js (hunk 3 above)
Comment (DanielMwendwa) [reply public/js/controllers/traffic_graphs_controller.js:1356]: I'll open another PR for this. It's quite different from the others.

File: public/js/controllers/traffic_graphs_controller.js (hunk 2 above)
Comment (DanielMwendwa) [reply public/js/controllers/traffic_graphs_controller.js:215]: The active/hovered value is clear.

**Pull request merged**